tests/transform-tests.js

```diff
@@ -690,4 +692,45 @@ describe("Transformations: ", function() {
       validateAns(result, original, '0.5.5');
     });
   });
+
+  describe("Upvert", function () {
+    it("should upert an ans types which has versions and typ and not in version_upverter_types", function () {
+      const version_types = [1, "0.10.10", "1.0.0", "0.4.5"]; 
+      for (const version of version_types) {
+        var original_ans = {
+          version: "0.10.9",
+          type: "story",
+          content_elements: [
+            {
+              embed: {
+                config: {
+                  editorState: {
+                    root: {
+                      type: "root",
+                      version: `${version}`,
+                    },
+                  },
+                },
+              },
+              subtype: "custom",
+              type: "custom_embed",
+            },
+          ],
+        };
+        var result = transforms.upvert(original_ans, LAST_VERSION);
+        assert.ok(result instanceof Error === false);
+        if (Version.is_semantic_version(version)) {
+          const expected = _.cloneDeep(original_ans);
+          expected.version = LAST_VERSION;
+          expected.content_elements[0].embed.config.editorState.root.version =
```

Comment:
This version shouldn't be modified.  It's the version of the config for the custom embed.  That's effectively an internal value for the custom embed.  We need to ensure this number is never modified by the upvert process.